src/components/@core/carousel/slide.tsx

```diff
@@ -33,14 +28,15 @@ const SlideDescription = ({ text }) =>
 
 const Slide = ({ resource, alt }) => {
   switch (resource.type) {
+    case ResourceType.Icon:
     case ResourceType.Image:
```

Comment:
i would be great to have a resource Interface for the resource props,Please make the same changes wherever resource props is used, for instance wherever "getResourceThumbnail" is used

Reply:
I don't know if I understood this correctly, since selection was near resource type switcher,

do you mean replace getObservationImage etc with getResourceThumbnail?

Reply:
updated